Add `TWC/2024`, `TWC 2024` registrations news post (#10977)
ppy/osu-wiki: wiki/Tournaments/TWC/2024/en.md

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1884428)
- [Livestream](https://www.twitch.tv/osulive)

## Ruleset

### Tournament rules

1. The osu!taiko World Cup is a country-based team tournament, played on the osu!taiko game mode.
   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted. Players that have participated under another country previously may continue to do so.
2. Beatmap scoring is based on ScoreV2.
3. The minimum size for a team is 4 players, and the maximum is 6.
4. The beatmap list for each round will be announced by the mappool selectors on the official stream, on the Sunday before the matches take place.
5. The match schedules for each round will be announced by the tournament managers on this page, as well as the information sheet, on the Sunday before the matches take place.

> 6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.

Reviewer: "use of the visual settings [...] is allowed"

```suggestion
6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
```

   - **Teams that do not not send their team list by March 8, 23:59 UTC will be disqualified from the competition.**
3. To ensure valid registrations, every prospective participant will be manually checked by the [account support team](/wiki/People/Account_support_team), in a manner similar to the [tournament screening](/wiki/Tournaments/Official_support#tournament-screening) that is offered to community tournaments.
   - Every registered user will be assigned to their respective country's candidate list.
   - To be successfully accepted on the list, players are required to be placed #5000 or higher on the osu!taiko global ranking, and not have violated the [osu! community rules](/wiki/Rules) within the last 12 months.
4. All successfully formed teams will be published after the registration phase on this page.
   - A team should have at least 2 players registered to have the chance to play the Qualifier stage.
5. Tournament staff members are **not** allowed to play in the tournament, with the exception of commentators and streamers.

### Qualifier instructions

1. In the Qualifier stage, all teams will play a specific pool designed by the mappool selectors.

> 2. The mappool will contain 5 brackets: [NoMod](/wiki/Gameplay/Game_modifier#nomod), [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), [Double Time](/wiki/Gameplay/Game_modifier/Double_Time), and [FreeMod](/wiki/Gameplay/Game_modifier#freemod). There will be no tiebreakers for this stage.

Reviewer: ```suggestion
2. The mappool will contain 5 brackets: [No Mod](/wiki/Gameplay/Game_modifier#nomod), [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), [Double Time](/wiki/Gameplay/Game_modifier/Double_Time), and [Free Mod](/wiki/Gameplay/Game_modifier#freemod). There will be no tiebreakers for this stage.
```

> 3. The Qualifier pool will consist of 7 maps. There will be 3 maps under the No Mod bracket and 1 map for the Hidden, Hard Rock, Double Time, FreeMod brackets.

Reviewer: ```suggestion
3. The Qualifier pool will consist of 7 maps. There will be 3 maps under the No Mod bracket and 1 map for the Hidden, Hard Rock, Double Time, and Free Mod brackets.
```

     - For a ready timer: the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
     - Repeat offenders may receive further sanctions from the tournament managers.
5. In the case of a team starting a map with less than the optimal number of players on their roster due to an unforeseen circumstance (e.g. a technical issue or unavoidable player absence), the team in question will be granted the right to fill their roster before play commences. The remainder of the original roster must stay the same, i.e. only player additions are allowed.
6. Each team will receive one "tactical timeout" of 2 minutes, to be used as extra time to ban, pick, or ready for a beatmap. *The tactical timeout is optional.*
   - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.

### Mappool information

1. Every stage will have its own mappool.

> 2. Each mappool consists of 5 brackets: [NoMod](/wiki/Gameplay/Game_modifier#nomod), [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), [Double Time](/wiki/Gameplay/Game_modifier/Double_Time) and [FreeMod](/wiki/Gameplay/Game_modifier#freemod).

Reviewer: ```suggestion
2. Each mappool consists of 5 brackets: [No Mod](/wiki/Gameplay/Game_modifier#nomod), [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock), [Double Time](/wiki/Gameplay/Game_modifier/Double_Time) and [Free Mod](/wiki/Gameplay/Game_modifier#freemod).
```

3. The mappool sizes are as follows:
   - Qualifiers: 10 beatmaps
   - Round of 32 and Round of 16: 14 beatmaps
   - Quarterfinals and Semifinals: 15 beatmaps
   - Finals and Grand Finals: 16 beatmaps
4. Each mappool has one tiebreaker, except for the Qualifiers.

> 5. The NoMod bracket will be played with the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod activated.

Reviewer: ```suggestion
5. The No Mod bracket will be played with the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod activated.
```

6. The [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Hard Rock](/wiki/Gameplay/Game_modifier/Hard_Rock) and [Double Time](/wiki/Gameplay/Game_modifier/Double_Time) brackets will be played with the respective mods activated along with the No Fail mod.

> 7. The [FreeMod](/wiki/Gameplay/Game_modifier#freemod) bracket will have Free Mods activated. The valid mod combinations for each player are:

Reviewer: ```suggestion
7. The [Free Mod](/wiki/Gameplay/Game_modifier#freemod) bracket will have Free Mods activated. The valid mod combinations for each player are:
```

>    - [NoMod](/wiki/Gameplay/Game_modifier#nomod) – no mods selected

Reviewer: ```suggestion
   - [No Mod](/wiki/Gameplay/Game_modifier#nomod), i.e. no mods selected
```

   - Hidden
   - Hard Rock
   - Hidden + Hard Rock
8. Failed scores will be counted. To that end, the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod will be enforced on every beatmap.
   - Teams that purposefully disregard this instruction and team fail as a result will lose said game.

> 9. The tiebreaker will be played under FreeMod conditions, but players will be exempt from the mod requirement.

Reviewer: ```suggestion
9. The tiebreaker will be played under Free Mod conditions, but players will be exempt from the mod requirement.
```

### Scheduling instructions

1. Each stage will be held on **a single weekend**.

> 2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each team may to choose a time slot in which they want to play.
>    - **A team that does not pick a time slot by March 14, 2024 (23:59 UTC) will be considered disqualified.**

Reviewer: ```suggestion
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each team may choose a time slot in which they want to play.
   - **Teams that do not pick a time slot by March 14, 2024 (23:59 UTC) will be considered disqualified.**
```
